Client/Application/AppDelegate.swift

```diff
@@ -158,11 +158,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
-        // We need to check if the app is a clean install to use for
-        // preventing the What's New URL from appearing.
-        if getProfile(application).prefs.intForKey(IntroViewControllerSeenProfileKey) == nil {
-            getProfile(application).prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
```

Comment:
Yea you should take it out. If we remove leanplum we want to make sure we dont accidently remove this as well